### ISSUE TITLE
AFKKickTask holds player instance incorrectly

### ISSUE THREAD

**Issue by Lucario2** [LegendOfMCPE/EssentialsPE#153]
```
PocketMine-MP Crash Dump Sat Aug 15 23:15:58 ACT 2015

Error: Call to a member function hasPermission() on null
File: /src/pocketmine/Player__32bit
Line: 439
Type: E_ERROR

Code:
[430] 		return $this->perm->isPermissionSet($name);
[431] 	}
[432] 
[433] 	/**
[434] 	 * @param permission\Permission|string $name
[435] 	 *
[436] 	 * @return bool
[437] 	 */
[438] 	public function hasPermission($name){
[439] 		return $this->perm->hasPermission($name);
[440] 	}
[441] 
[442] 	/**
[443] 	 * @param Plugin $plugin
[444] 	 * @param string $name
[445] 	 * @param bool   $value
[446] 	 *
[447] 	 * @return permission\PermissionAttachment
[448] 	 */
[449] 	public function addAttachment(Plugin $plugin, $name = \null, $value = \null){

Backtrace:
#0 /src/pocketmine/Server__32bit(0): pocketmine\Server->crashDump()
#1 C:/Users/eliott/Desktop/PockeMine/Plugins/EssentialsPE_v1.1.2/src/EssentialsPE/BaseFiles/BaseSession(66): pocketmine\Player->hasPermission()
#2 C:/Users/eliott/Desktop/PockeMine/Plugins/EssentialsPE_v1.1.2/src/EssentialsPE/Loader(832): EssentialsPE\BaseFiles\BaseSession->__construct()
#3 C:/Users/eliott/Desktop/PockeMine/Plugins/EssentialsPE_v1.1.2/src/EssentialsPE/Loader(873): EssentialsPE\Loader->createSession()
#4 C:/Users/eliott/Desktop/PockeMine/Plugins/EssentialsPE_v1.1.2/src/EssentialsPE/Loader(894): EssentialsPE\Loader->getSession()
#5 C:/Users/eliott/Desktop/PockeMine/Plugins/EssentialsPE_v1.1.2/src/EssentialsPE/Tasks/AFK/AFKKickTask(27): EssentialsPE\Loader->isAFK()
#6 /src/pocketmine/scheduler/TaskHandler(150): EssentialsPE\Tasks\AFK\AFKKickTask->onRun()
#7 /src/pocketmine/scheduler/ServerScheduler(247): pocketmine\scheduler\TaskHandler->run()
#8 /src/pocketmine/Server__32bit(2494): pocketmine\scheduler\ServerScheduler->mainThreadHeartbeat()
#9 /src/pocketmine/Server__32bit(2300): pocketmine\Server->tick()
#10 /src/pocketmine/Server__32bit(2178): pocketmine\Server->tickProcessor()
#11 /src/pocketmine/Server__32bit(1778): pocketmine\Server->start()
#12 /src/pocketmine/PocketMine(464): pocketmine\Server->__construct()
#13 (1): ()
#14 (0): {main}()
```

**Comment by DinoKill** [comment]
Are you trying to do this in console?

**Comment by Lucario2** [comment]
No.

**Comment by Lucario2** [comment]
They can't fix anything.

**Comment by Lucario2** [comment]
They just close my problems and threaten to ban me.

**Comment by Lucario2** [comment]
I want my problems to be fixed do they even ask me whats wrong or why isnt it working?

**Comment by LDX-MCPE** [comment]
Other people have already reported that issue. We don't want to try to have people telling us if we fixed it or not in six seperate issues.

**Comment by Lucario2** [comment]
The problem is with the afk kick task. Is there a if player isnt on don't crash event? (LOL)

**Comment by iksaku** [comment]
We are searching the origin of this problem, and will be fixed in the next version... But don't expect it to be released soon

**Comment by PEMapModder** [comment]
@LDX-MCPE this is not a duplicated issue. This is about AFKKickTask, the other one was about TPRequestTask.

**Comment by LDX-MCPE** [comment]
@PEMapModder "...already reported *that* issue..." @Lucarlo2 started talking about another issue in this issue's comments after I locked the other issue.

**Comment by Falkirks** [comment]
I pushed a quick fix for this in c99f97dd6262d7386acf32c3cc90f2aa59705afa. Still needs testing.
